softserve/views.py

```diff
@@ -80,35 +70,48 @@ def dashboard():
 
 
 @app.route('/create_node', methods=['GET', 'POST'])
-#@organization_access_required('gluster')
+@organization_access_required('gluster')
 def get_node_data():
-    count = db.session.query(func.count(Vm.id)) \
-            .filter_by(state = 'ACTIVE').scalar()
-    if count >=5:
-        flash('Limit got over!')
-        return render_template('home.html')
+    if request.method == "POST":
+        counts = request.form['counts']
+        name = request.form['node_name']
+        hours_ = request.form['hours']
+        pubkey_ = request.form['pubkey']
+
+        '''Validating the SSH public key'''
+        ssh = sshpubkeys.SSHKey(pubkey_, strict=True)
+        try:
+            ssh.parse()
+        except Exception:
```

Comment:
Do not catch all exceptions. Figure out what exception this raises. And log the exception.